runtime/component/app-scheduler.js

```diff
@@ -74,6 +78,7 @@ AppScheduler.prototype.appLauncher = {
  * @param {string} appId
  * @param {string | number} mode
  * @param {object} [options]
+ * @param {'light' | 'exe' | 'hive' |'default'} [options.type] - app type to be used
```

Comment:
```suggestion
 * @param {'light' | 'exe' | 'hive' | 'default'} [options.type] - app type to be used
```

Reply:
Will change the default to hive?

Reply:
Tracking of make hive the default launcher would be on #899